back/engines/commercial/id_vienna_saml/app/lib/id_vienna_saml/citizen_saml_omniauth.rb

```diff
@@ -36,6 +36,11 @@ def profile_to_user_attrs(auth)
       }
     end
 
+    def profile_to_user_attrs_for_update(auth)
+      # Don't update first and last name which are probably generated from email
```

Comment:
What do you mean by "probably"? Generated by the provider or by us?

Reply:
By us in `generate_placeholder_name_from_email` when names provided by SSO are blank.